util/chunk/chunk.go

```diff
@@ -38,6 +38,7 @@ type Chunk struct {
 // Capacity constants.
 const (
 	InitialCapacity = 32
+	NoDataChunkCap  = -1
```

Comment:
how about:
```go
ZeroCapacity = 0
```

Reply:
`0` is ok, but it will still alloc columns and field num of column.. use magic -1 we can alloc less.